src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/config/ResourceServerConfiguration.kt

```diff
@@ -24,8 +24,7 @@ class ResourceServerConfiguration : WebSecurityConfigurerAdapter() {
         auth.antMatchers(
           "/webjars/**", "/favicon.ico", "/csrf",
           "/health/**", "/info",
-          "/v3/api-docs/**",
-          "/swagger-ui/**"
+          "/v3/api-docs/**", "/swagger-ui/**", "/swagger-ui.html"
```

Comment:
missed out swagger-ui.html so we don't get the redirect

Reply:
As saw that yesterday it wasn;t working